Airdnd_Back/src/main/java/com/airdnd/back/SearchController.java

```diff
@@ -121,17 +121,55 @@ public String check(HttpServletRequest request, HttpServletResponse response, @R
 				typeshared1 = "다인실";
 				typeshared2 = "객실";
 			}
-			
+
 			Map<Integer, String> hostlangmap = new HashMap<Integer, String>();
 			for(int i = 0; i < 5; i++) hostlangmap.put(i, "devengerslang");
-			
+
 			if(hostLangList.equals("0")) for(int i = 0; i < 5; i++) hostlangmap.put(i, "");
 			else {
 
 				String[] hostlang = hostLangList.replace("중국어", "中文").replace("영어", "English").replace("프랑스어", "Français").replace("스페인어", "Español").split("-");
 				for(int i = 0; i < hostlang.length; i++) hostlangmap.replace(i, hostlang[i]);
 			}
-			
+
+			String[] amenityListArr = amenityList.split("-");
+			String[] facilityListArr = facilityList.split("-");
+			String[] ListArr = new String[20];
+			System.arraycopy(amenityListArr, 0, ListArr, 0, amenityListArr.length);
+			System.arraycopy(facilityListArr, 0, ListArr, amenityListArr.length, facilityListArr.length);
+
+			String queryFirst = " and (facility like '%";
+			String queryMiddle = "%' or facility like'%";
+			String queryLast = "%')";
+			String query = queryFirst;
+
+			outer : for(int i = 0; i < ListArr.length; i++) {
+				
+				if(ListArr[i].equals("0") && ListArr[i+1].equals("0")) {// 둘다 없을때
+					query = "";
+					break outer ;
+				}else if(!(ListArr[0].equals("0")) && !(ListArr[i].equals("0")) && ListArr[i+1] == null) { //fac, ame 다 있는 마지막
+					query += ListArr[i] + queryLast;
+					System.out.println("다있는 마지막 " + query);
+					break outer;
```

Comment:
When there are both filters in param

Reply:
It is good to delete "system.out.println" because it is slow 👍